src/app/recent/nearby/map-nearby.tsx

```diff
@@ -44,14 +61,35 @@ function BirdCard({
     <Card
       ref={cardRef}
       onClick={onClick}
-      className={cn("h-full whitespace-nowrap p-4", {
+      className={cn("h-full min-w-64 whitespace-nowrap p-4", {
         "border-2 border-orange-400": isSelected,
       })}
     >
-      <div className="text-base font-bold">{sighting.comName}</div>
+      <div className="space-between flex flex-row items-center justify-between gap-2">
+        <div className="text-base font-bold">{sighting.comName}</div>
+        {isLoading ? (
+          <div className="h-8 w-8 animate-pulse rounded-sm bg-gray-300" />
+        ) : birdImage ? (
+          <Image
```

Comment:
I think another good refactor target here would be to pull this image into its own component, then you can have a `src` prop that can be `undefined` and inside the component you can either return the placeholder or the image itself. Then the image doesn't need to care about how to get the `src` or even what it is, just whether it exists or not.